user/pages/04.Reference/08.network/docs.en.md

```diff
@@ -1,45 +1,45 @@
 ---
 title: 'Network'
 published: true
-date: '26-07-2021 16:25'
+date: '25-11-2021 10:15'
 taxonomy:
     category:
         - docs
 ---
 
 ## Overview
 
-Using the SysEleven Stack Network service, you can do much more than just accessing the internet; It's possible to build arbitrary private network topologies using virtual routers, networks, subnets, load balancers and VPN site connections.
+Using the SysEleven Stack Network service, you can do much more than just accessing the Internet; It's possible to build arbitrary private network topologies using virtual routers, networks, subnets, load balancers and VPN site connections.
 
-Servers can access the internet using virtual routers (SNAT). For making servers and load balancers accessible from the internet, we offer a Floating IP service (DNAT).
+Servers can access the Internet using virtual routers (SNAT). For making servers and load balancers accessible from the Internet, we offer an IPv4 Floating IP service (DNAT), Load Banacers and IPv6 Global Unicast Addresses.
 
 You can improve security and control isolation by using security groups.
 
-You can manage networking objects both via our [public OpenStack API endpoints](../../02.Tutorials/02.api-access/docs.en.md), as well as using the [Dashboard](https://dashboard.cloud.syseleven.net).
+You can manage networking objects both via our [public OpenStack API endpoints](../../02.Tutorials/02.api-access/docs.en.md), as well as using the [Dashboard](https://cloud.syseleven.de/).
```

Comment:
There are even more of those old URLs throughout the docs. Luckily, we have a redirect. Shall we fix them all, again, for consistency?